pom.xml

```diff
@@ -122,17 +122,12 @@
   </ciManagement>
 
   <properties>
-    <mojo.java.target>8</mojo.java.target>
-
     <!-- some of plugins requires JDK 11 -->
     <recommendedJavaBuildVersion>11</recommendedJavaBuildVersion>
-
-    <junitBomVersion>5.10.0</junitBomVersion>
-    <maven.version>3.2.5</maven.version>
```

Comment:
This one change - requirement Maven from 3.2.5 to 3.6.3 should be in separate PR in order to show in release notes

Reply:
README also to fix with new Maven requirement